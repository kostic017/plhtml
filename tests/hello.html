<!doctype html>
<html lang="en">
    <head>
        <title>`Hello World`</title>
    </head>
    <body>

        <main>

            <output>`Enter your name: `</output>

            <var class="string">str</var>
            <input name="str">

            <output>`Hello ` + str + `!`</output>

        </main>

    </body>
</html>
