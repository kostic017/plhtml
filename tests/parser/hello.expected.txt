"Hello World"
Main:
    Write: "Enter your name: "
    string str
    Read str
    Write: "Hello " + str + "!"
